rise-google-sheet.html

```diff
@@ -237,17 +237,49 @@
       },
 
       _prepareResponse: function (resp) {
-        var response = {};
+        var response = {},
+          firstRowCols = 0,
+          totalCols = 0;
 
         /*
          Provide an empty array if values property missing in response object. This can occur if any range
          values (minColumn, maxColumn, minRow, maxRow) are out of scope of the data entered in worksheet
         */
         response.results = (resp.values) ? resp.values : [];
 
+        // Workaround for this issue - https://goo.gl/rsazDP.
+        firstRowCols = (response.results && response.results.length > 0) ? response.results[0].length : 0;
+        totalCols = this._getTotalCols(response.results);
+
+        // Append cells to the first row until it matches the total column count.
+        if (firstRowCols < totalCols) {
```

Comment:
@stulees It actually doesn't appear to be necessary. I just tested with several rows of data, all of which have nothing in the last column except the last row, and the last column showed just fine. The Widget likely uses the first row as the defining structure.